plugin/pprof/pprof.go

```diff
@@ -31,6 +31,7 @@ func (h *handler) Startup() error {
 	h.ln = ln
 
 	h.mux = http.NewServeMux()
+	h.mux.HandleFunc(path, redirectTo(path+"/"))
```

Comment:
I can see this being handy, but a whole `redirectTo` function seems overkill. I would just inline the handler directly here.

Reply:
Done, thanks! 

> I can see this being handy, but a whole `redirectTo` function seems overkill. I would just inline the handler directly here.